jobs/MasterCI/MasterCI

```diff
@@ -1,42 +1,60 @@
 node{
-    deleteDir()
-    checkout scm
-    def shareMethod = load("jobs/ShareMethod.groovy")
-    try{
-        stage("Create Manifest"){
-            load("jobs/create_manifest/create_manifest.groovy")
-        }
-        def manifest_name=env.MANIFEST_FILE_URL.tokenize('/')[-1];
-        currentBuild.description = "<a href=${env.MANIFEST_FILE_URL}>${manifest_name}</a>";
+    withEnv([
+        "branch=${env.branch}",
+        "IS_OFFICIAL_RELEASE=false",
+        "date=current",
```

Comment:
are you remove "date"/"timezone" from Jenkins build parameter ?
and what's the benifit of repeat the ENV Variable here? as a double check in case the ENV Var(Build Param) not given ?